tests/test_plugin_vk.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+
+from streamlink.plugins.vk import VK
+
+
+class TestPluginStreamMe(unittest.TestCase):
+    def test_follow_vk_redirect(self):
```

Comment:
Ooof, and now I see that I forgot to commit the last set of changes to vk.py, sorry about that.